### ISSUE TITLE
NamedArguments: Exclude Lambdas

### ISSUE THREAD

**Issue by shivangtripathii** [detekt/detekt#4999]
The rule or changes you're looking for might have already been suggested!
Please search in the [issues](https://github.com/detekt/detekt/issues) before creating a new one.

## Expected Behavior of the rule

When I configure the rule `NamedArguments` with threshold: 1 I start getting warnings for lambda functions as well
`Named arguments are required for function invocation with many parameters.`

```
require(n == 2) {
  "N is not 2"
}
```

## Context

Can we have a configuration to ignore lambdas?

**Comment by marschwar** [comment]
I agree, that this does not feel right. But it may not be as easy as that. Consider a function that takes multiple lambdas as arguments. In such a case I want (most likely) all of them to named.

```kotlin
fun doSomething(onSuccess: (String) -> Unit, onError: (Exception) -> Unit) {
    // ...
}
```

I think the rule should exclude a single trailing lambda from counting towards the threshold. 
I vote to change the rule without an additional config parameter.

**Comment by BraisGabin** [comment]
We could have a new rule to avoid trailing lambda notation in cases that there are multiple lambdas at the end... But for this rule I agree that the trailing lambda shouldn't count towards the threshold.
